cicd/pipelines/using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: ASSEMBLY
+[id="using-pipelines-as-code"]
+= Using Pipelines as Code
+include::_attributes/common-attributes.adoc[]
+:context: using-pipelines-as-code
+
+toc::[]
+
+:FeatureName: Pipelines as Code
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+Pipelines as Code allows cluster administrators and users with necessary privileges to define pipeline templates as part of source code Git repositories. When triggered by source code push or a pull request for the configured Git repository, it runs the pipeline and reports status.  
```

Comment:
```suggestion
With Pipelines as Code, cluster administrators and users with necessary privileges can define pipeline templates as part of source code Git repositories. When triggered by source code push or a pull request for the configured Git repository, it runs the pipeline and reports status.  
```
- Per IBM SG, avoid giving software human characteristics with words like "allow". See the section on "Anthropomorphism": "Do not assign human characteristics to inanimate objects, which are incapable of human behaviors and emotions. As much as possible, focus technical information on users and their actions, not on a product and its actions."